widget/playground/src/store/config.ts

```diff
@@ -13,15 +13,7 @@ import { getConfig } from '../configs';
 import createSelectors from './selectors';
 
 export type Mode = 'dark' | 'light' | 'auto';
-export type COLORS =
-  | 'background'
-  | 'primary'
-  | 'success'
-  | 'error'
-  | 'warning'
-  | 'surface'
-  | 'neutral'
-  | 'foreground';
+export type COLORS = 'background' | 'primary' | 'secondary' | 'foreground';
```

Comment:
Isn't this variable duplicate? (check config.ts changes)

Reply:
fixed